src/main/scala/cromwell/binding/expression/DeclarationEvaluator.scala

```diff
@@ -0,0 +1,14 @@
+package cromwell.binding.expression
+
+import cromwell.binding.Declaration
+import cromwell.parser.WdlParser.AstNode
+
+import scala.language.postfixOps
+import scala.util.Try
+
+case class DeclarationEvaluator(override val lookup: String => Seq[Declaration], override val functions: WdlFunctions[Seq[Declaration]]) extends Evaluator {
+  override type T = Seq[Declaration]
+
+  override def evaluate(ast: AstNode): Try[Seq[Declaration]] = ???
+}
+
```

Comment:
Oh yeah...  I just didn't have time to implement it.  I should remove this.  It was going to be used in the code that decides localization strategies